Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/04-network-hub.md

```diff
@@ -70,6 +70,10 @@ After the domain controller is up and running, you will need to update the DNS o
    az vm restart -g "aks-HUB" -n "server-dev-linux"
    ```
 
+### Windows client for system to manage the cluster
+04-Network-Hub will execute one client system as well to run the admin and operational tasks of the AKS cluster. Sub module (compute-Win) will have a client system ready with min required tools such as AzCli, Kubectl etc.
+To connect client system you can use Bastion service and provide the VM name or IP address in bastion service. Username and Password for the client will be used from variable.tf. 
```

Comment:
It's not clear if these are instructions or just notes.  Where you put this was in the path for step-by-step instructions.  If these are just notes, you'll need to find another way to make these not feel like they are part of the steps in this deployment process or find another place to put these notes.  How about simply using the new jumbbox in the instructions (there is a place already where it says to connect to the jumpbox -- just update there to use this new infra.)

Reply:
I thought writing a note will be enough for the end user to consume the service, this note educates that there is a windows VM client which can be used through the Bastion service.
We can add a snapshot of using Bastion service utilizing the Windows VM client. That should be enough for how to.

Reply:
How to diagram added.

Reply:
The feedback was though that you put this "note" right in the middle of the instructions.  So it's not clear to a user if this is supposed to be an action they take or this is just some information they need to memorize for a future step or whatever.  We don't want to mix learning modalities here.  if this is part of the instructions, make it read like instructions.  If this is notes, please find a spot that isn't part of the instructions -- or find the spot in the instructions that already tell you to connect to the jumpbox and include content like this there.  There are instructions telling a user to do that, so that would be a better place for this.

Reply:
Updated the section for just a note for the windows client. Users to get a note of the IP address of the client machine for later use. I hope now it's not disturbing the flow of the instructions.